deploy/templates/nstemplatetiers/appstudio/ns_appstudio.yaml

```diff
@@ -131,7 +131,7 @@ objects:
 - apiVersion: rbac.authorization.k8s.io/v1
   kind: RoleBinding
   metadata:
-    name: ${USERNAME}-sa-read
+    name: appstudio-sa-read
```

Comment:
```suggestion
    name: member-operator-sa-read
```

Reply:
I thought we would use `appstudio-` for all role bindings related to AppStudio, but I don't have any strong opinion on the names or prefixes.

Reply:
ok, got it. As I said, I'm fine with both options ;)